Whole-game.Rmd

```diff
@@ -54,51 +60,61 @@ Do you have an old version of devtools? Compare your version against ours and up
 packageVersion("devtools")
 ```
 
-For presentation purposes only, we use [fs](https://fs.r-lib.org), for filesystem work, and the [tidyverse](https://tidyverse.tidyverse.org), for light data wrangling.
-
-```{r, R.options = list(tidyverse.quiet = TRUE)}
-library(tidyverse)
-library(fs)
-```
-
-## Toy package: foofactors
+## Toy package: regexcite
 
 We use various functions from devtools to build a small toy package from scratch, with features commonly seen in released packages:
 
-  * Functions to address a specific need, such as helpers to work with factors.
-  * Access to established workflows for installation, getting help, and checking
-    basic quality.
-  * Version control and an open development process.
-    - This is completely optional in your work, but recommended. You'll see how
-      Git and GitHub helps us expose all the intermediate stages of our package.
-  * Documentation for individual functions via
-    [roxygen2](https://CRAN.R-project.org/package=roxygen2).
-  * Unit testing with [testthat](https://testthat.r-lib.org).
-  * Documentation for the package as a whole via an executable `README.Rmd`.
+* Functions to address a specific need, in this case helpers for work with regular expressions.
+* Version control and an open development process.
+  - This is completely optional in your work, but highly recommended.
+    You'll see how Git and GitHub help us expose all the intermediate stages of
+    our toy package.
+* Access to established workflows for installation, getting help, and checking quality.
+  - Documentation for individual functions via [roxygen2](https://CRAN.R-project.org/package=roxygen2).
+  - Unit testing with [testthat](https://testthat.r-lib.org).
+  - Documentation for the package as a whole via an executable `README.Rmd`.
 
-We call the package __foofactors__ and it will have a couple functions for handling factors. Please note that these functions are super simple and definitely not the point! For a proper package for factor handling, please see [forcats](https://forcats.tidyverse.org).
+We call the package **regexcite** and it will have a couple functions that make common tasks with regular expressions easier.
```

Comment:
It doesn't really have a "couple" functions, it's just one. But I still think this wording is OK. Or, if we started to suggest exercises for the reader, we could certainly suggest another function or two for them to practice adding to the package (complete with roxygen comment, tests, inclusion in NAMESPACE, mention in README, etc.).